frontend/src/features/documents/list/DocumentListView.tsx

```diff
@@ -38,8 +39,8 @@ export const DocumentListView: React.FunctionComponent<IDocumentListViewProps> =
         documentItems = documentItems.filter(document => {
           return (
             (!filters.documentTypeId ||
-              document.documentTypeId === Number(filters.documentTypeId)) &&
-            (!filters.status || document.statusTypeCode?.id === filters.status)
+              document?.documentType?.id === Number(filters.documentTypeId)) &&
+            (!filters.status || document?.statusTypeCode?.id === filters.status)
```

Comment:
The type was not flattened on the backend, but it was on the front end. This fixes that